res/controllers/lodash.mixxx.js

```diff
@@ -8,6 +8,12 @@
  * Copyright Jeremy Ashkenas, DocumentCloud and Investigative Reporters & Editors
  */
 ;(function() {
+  const DEPRECATION_MSG = (
+      "Lodash is deprecated and will be removed in Mixxx 2.6.0; refer to the page " +
```

Comment:
```suggestion
      "Lodash is deprecated and will be removed in a future release of Mixxx; refer to the page " +
```
I'm in favor of deprecating Lodash and printing a warning, but we should not remove it to early. Third party mappings might break due to this change.

Reply:
Done. Thank you.